akka-sample-distributed-data-java/src/main/java/sample/distributeddata/ShoppingCart.java

```diff
@@ -71,6 +82,7 @@ public Cart(Set<LineItem> items) {
     public final String title;
     public final int quantity;
 
+    @JsonCreator
```

Comment:
this shouldn't be needed, but you should add `"-parameters"` to `javacOptions`

`@JsonCreator` is normally only needed when there is a single parameter

Reply:
the jackson issue is solved when adding `"-parameters"`  to `javacOptions`